boilerplate/e2e/README.md

```diff
@@ -0,0 +1,66 @@
+# Detox End-To-End Testing
+
+## Setup
+
+To get your Detox tests up and running, you'll need to install some global dependencies:
+
+1. Install the latest version of [Homebrew](https://brew.sh/)
+2. Make sure you're running node 8.6.0 or higher. If you aren't, you can run:
+
+```bash
+nvm install 8.6.0
```

Comment:
Let's recommend latest here rather than 8.6.0.

Reply:
8.6.0 came from what's currently in Solidarity (and Detox itself only required 8.3.0), but I can try it out with the latest.

Reply:
I'm actually on the fence about this. If there's no reason that 8.6 doesn't work at a minimum, what's the value behind recommending higher?

Reply:
In my experience, people will install (by rote) whatever you tell them to install. If they're installing a new Node, might as well make sure they're on a new one. We could put a note that we support down to `8.6.0` but recommend latest.

Reply:
Yeah, that's a good point. I won't change solidarity, but I'll update this to be `latest`

Reply:
Possibly related: https://github.com/infinitered/ignite/issues/1358#issuecomment-458075016

Reply:
Hah, so Node 8 works, and node 10 works but not 9? That's... ugh.